drizzle/migrations/0007_add_chat_contexts.sql

```
@@ -0,0 +1,73 @@
+BEGIN
+    IF NOT EXISTS (
+        SELECT 1 FROM information_schema.table_constraints
+        WHERE constraint_name = 'chat_contexts_chat_id_chats_id_fk'
+    ) THEN
+        ALTER TABLE "chat_contexts" ADD CONSTRAINT "chat_contexts_chat_id_chats_id_fk" FOREIGN KEY ("chat_id") REFERENCES "chats"("id") ON DELETE cascade;
+    END IF;
+END $$;
+
+-- Migrate existing drawing-context role:'data' rows from messages into the new table
+INSERT INTO "chat_contexts" ("chat_id", "data", "created_at", "updated_at")
+SELECT "chat_id", "content"::jsonb, "created_at", "created_at"
```

SQLFluff (4.2.2)

[error] Reuse of column alias '"created_at"' from line 24.

(AL08)